docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -68,17 +68,16 @@ Running dual-region setups requires the users to be able to detect any regional
 
 ## Procedure
 
-We don't differ between active and passive regions as the procedure is the same for either loss. We will focus on losing the passive region while still having the active region.
+We handle the loss of both active and passive regions using the same procedure. For clarity, this section focuses on the scenario where the passive region is lost while the active region remains operational.
 
-You'll need to reroute the traffic to the surviving region with the help of DNS (details on how to do that depend on your DNS setup and are not covered in this guide.)
+#### Key Steps to Handle Passive Region Loss
 
-After you've identified a region loss and before beginning the region restoration procedure, ensure the lost region cannot reconnect as this will hinder a successful recovery during failover and failback execution.
-
-In case the region is only lost temporarily (for example, due to network hiccups), Zeebe can survive a region loss but will stop processing due to the loss in quorum and ultimately fill up the persistent disk before running out of volume, resulting in the loss of data.
-
-The **failover** phase of the procedure temporarily restores Camunda 8 functionality by removing the lost brokers and the export to the unreachable Elasticsearch instance.
-
-The **failback** phase of the procedure results in completely restoring the failed region to its full functionality. It requires you to have the lost region ready again for the redeployment of Camunda 8.
+1. **Traffic Rerouting:** Reroute traffic to the surviving active region using DNS. (Details on how to manage DNS rerouting depend on your specific DNS setup and are not covered in this guide.)
+2. **Prevent Reconnection:** Ensure that the lost region cannot reconnect before starting the restoration procedure. Reconnection could interfere with a successful recovery during failover and failback.
```

Comment:
you can cut this point 2. out, it's not applicable for the new procedure anymore.